src/miner_lora.erl

```diff
@@ -54,7 +54,11 @@
     reg_throttle = undefined :: undefined | miner_lora_throttle:handle(),
     last_tmst_us = undefined :: undefined | integer(),  % last concentrator tmst reported by the packet forwarder
     last_mono_us = undefined :: undefined | integer(),  % last local monotonic timestamp taken when packet forwarder reported last tmst
-    chain = undefined :: undefined | blockchain:blockchain()
+    chain = undefined :: undefined | blockchain:blockchain(),
+    radio_udp_bind_ip,
+    radio_udp_bind_port,
+    cur_poc_challenger_type = undefined :: undefined | validator,
+    following_chain = true :: boolean() | undefined
```

Comment:
Nothing seems to set this, but it is checked for being false?

Reply:
https://github.com/helium/miner/pull/1068/commits/32247902ed59dcdcf354746ce707c1a1aa4c5eda